platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/storage/FileDirsPathsAsync.kt

```diff
@@ -0,0 +1,26 @@
+package org.maplibre.android.storage
+
+import android.content.Context
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+fun fileDirsPathsAsync(
```

Comment:
Does this need to be public?

Reply:
Again, it's internal to the package.

Reply:
I think it is public now (as that is the default) and should be marked `internal`?

Reply:
Fixed.